src/common/hooks/useZeroApprovalState.ts

```diff
@@ -0,0 +1,16 @@
+import { Currency } from '@uniswap/sdk-core'
+import { atom, useAtomValue } from 'jotai'
+
+interface ZeroApprovalState {
+  isApproving: boolean
+  currency?: Currency
+}
+
+export const zeroApprovalState = atom<ZeroApprovalState>({
```

Comment:
This file is supposed to be located in `state` directory

Reply:
Ah! Good one!

Reply:
The file should be named as `atoms.ts` or `zeroApprovalStateAtom.ts` :)
Actually, we still don't have a convention on this, so, feel free to ignore